plugins/utils/baleia/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  helpers,
+  pkgs,
+  config,
+  ...
+}: let
+  mkLogOption = helpers.defaultNullOpts.mkEnum ["ERROR" "WARN" "INFO" "DEBUG"];
+in
```

Comment:
As you only use this once, I don't really see a point in isolating it in a `let` `in` block.

Reply:
and done =)